app/controllers/base_controller.rb

```diff
@@ -115,4 +122,6 @@ def before_update_attributes; end
   def destroy_resource
     resource.destroy
   end
+
+  def parent_resource; end
```

Comment:
@samnang does this need to be here?

Reply:
I think it's just a place holder because I see the pattern in both api and dashboard load the parent resource.